artifacts/ArtifactHandling/Add-ArtifactsLog.ps1

```diff
@@ -18,7 +18,9 @@ function Add-ArtifactsLog {
         [System.Object]$data = $null,
         [Parameter(Mandatory=$false)]
         [string]$artifactsLogFile = "C:/inetpub/wwwroot/http/artifacts.log.json",
-        [switch]$lowerCase
+        [switch]$lowerCase,
+        [string]$suppressedWarnings = $env:SUPPRESSED_WARINGS,
```

Comment:
typo in `SUPPRESSED_WARINGS` -> Warnings

Reply:
@lippertmarkus fixed